File: FN4RetornoWin/Form1.vb
Public Class Form1

    Dim mon As FN4RetornoCtl.RetornoMonitor

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        mon = New FN4RetornoCtl.RetornoMonitor
        mon.run()
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        mon.pause()
    End Sub
End Class

File: FN4InutilizacaoWin/Form1.cs
using System.Windows.Forms;
using FN4InutilizacaoCtl;

namespace FN4InutilizacaoWin
{
    public partial class Form1 : Form
    {
        private InutilizacaoMonitor _mon = new InutilizacaoMonitor();

        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, System.EventArgs e)
        {
            _mon.Run();
        }

        private void button2_Click(object sender, System.EventArgs e)
        {
            _mon.Pause();
        }
    }
}
